src/GraphQL.DataLoader.Tests/Types/OrderItemType.cs

```diff
@@ -14,8 +14,7 @@ public OrderItemType(IDataLoaderContextAccessor accessor, IProductsStore product
         Field(x => x.Quantity);
         Field(x => x.UnitPrice);
 
-        Field<ProductType, Product>()
-            .Name("Product")
+        Field<ProductType, Product>("Product")
```

Comment:
It's not related directly to my changes, I just stop using obsoleted overload in some places.